packages/components/src/alert/src/Alert.css

```diff
@@ -1,7 +1,29 @@
-.o-ui-alert-warning-icon {
-    color: var(--hop-warning-icon-weak);
+.o-ui-alert-button-group > .o-ui-button {
+    max-width: 50%;
```

Comment:
I'm pretty sure this alert modal can have 3 buttons. I'm guessing this style breaks with 3 buttons

Reply:
That one issue I raised with design regarding the current design, apparently we will never have 3 buttons.

Reply:
The code still support it at the moment.

Reply:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/024c9328-9962-41af-acfd-ac7dce350a89)

Forgot it was working with 3 buttons.